### ISSUE TITLE
Look into Layer Normalization

### ISSUE THREAD

**Issue by soi** [weberlab-hhu/GeenuFF#97]
"Empirically, we showed that recurrent neural networks benefit the most from the proposed method
especially for long sequences and small mini-batches."

https://arxiv.org/pdf/1607.06450.pdf
https://mlexplained.com/2018/01/13/weight-normalization-and-layer-normalization-explained-normalization-in-deep-learning-part-2/
https://pypi.org/project/keras-layer-normalization/

**Comment by soi** [comment]
Seems to help a bit with shifting the prediction towards a better genic performance (at cost of intergenic predictions, which is what we want)
